### DISCUSSION TITLE
Is there a way to view the federated timeline?

### DISCUSSION THREAD

**Discussion by mojosam** [bookwyrm-social/bookwyrm#2105] [General]
Is there a way to view the federated timeline (like I can do on Mastodon)?

I'm new to Bookwyrm. Maybe my instance just doesn't have that enabled?

**Comment by mouse-reeve** [comment]
Short answer is no, longer answer is maybe, kind of, in a way, someday.

The longest answer is that BookWyrm doesn't use the same timeline system as Mastodon -- instead we have three status views: the home timeline, which is analogous to Mastodon's, the Books feed, which shows you statuses related to books on your shelves, and the "Discover" view, which shows the type of statuses you'd find on Mastodon's local timeline, but with a different interface and different options for how to interact with those statuses.

There is a ticket open for creating a federated timeline version of the discover view (#1473), which would do something akin to Mastodon's federated timeline.

**Reply by mojosam** [reply]
I hope that gets implemented in the foreseeable future. I'm on a smaller instance. It would be good to be able to see what's happening in the wider Bookwyrmosphere.
